docs/data-formats.md

```diff
@@ -528,22 +528,38 @@ attributes are applied.
 <attribute fullname="SomecustomAttribute" feature="EnableOptionalFeature" featurevalue="false"/>
 ```
 
-### Special custom attributes
+### Removing custom attributes
 
-Also if the attribute is used in a type, a special property can be used to specify that the type
-is a Custom Attribute and it's instances should be removed by the linker. To do this use `internal="RemoveAttributeInstances"` instead of specifying `fullname` in the attribute as described in the following
-example:
+Any custom attribute can be annotated with a special custom attribute which can be used to specify
+that all instances of the attribute can be removed by the linker. To do this use `internal="RemoveAttributeInstances"`
+instead of specifying `fullname` in the attribute as described in the following example:
 
 ```xml
 <linker>
-  <assembly fullname="*"> 
+  <assembly fullname="*">
     <type fullname="System.Runtime.CompilerServices.NullableAttribute">
       <attribute internal="RemoveAttributeInstances" feature="EnableOptionalFeature" featurevalue="false" />
     </type>
   </assembly>
 </linker>
 ```
 
+In some cases it's useful to remove only specific instances of same attribute. This can be achieved by specifying values of the arguments to match
```

Comment:
This should say that the sample will only remove `[AssemblyMetadataValue("RemovableValue")]` but leave alone for example `[AssemblyMetadataValue("KeptValue")]`.

Reply:
Updated the explanation